src/erlcloud_aws.erl

```diff
@@ -822,9 +822,16 @@ get_host_vpc_endpoint(Service, Default) when is_binary(Service) ->
     %% resolve through ENV if any
     Endpoints = case ConfiguredEndpoints of
         {env, EnvVarName} when is_list(EnvVarName) ->
-            Es = string_split(os:getenv(EnvVarName, ""), ","),
             % ignore "" env var or ",," cases
-            [list_to_binary(E) || E <- Es, E /= ""];
+            % also handle "zoneID:zoneName" form when it comes from CFN
+            Es = string_split(string_split(os:getenv(EnvVarName, ""), ","), ":"),
```

Comment:
we need to `split_string(':')` every string returned by `split_string(',')`.

Reply:
oh!

Reply:
I mean `string_split()` returns a list of strings and doesn't accept a list of strings as input.

Reply:
fixed